_integration-schemas/pendo/v1/poll_events.md

```diff
@@ -13,7 +13,7 @@ description: |
 
 replication-method: "Key-based Incremental"
 replication-key:
```

Comment:
This is actually an attribute that will be in the table. `replication-key.name` as you have it here should only be used when this is *not* the case. 

[Here's the schema ](https://github.com/bytecodeio/tap-pendo-release/blob/05a2bd778230fba39d1a9671004ef9d7efb2418f/tap_pendo/schemas/poll_events.json#L27)for `poll_events`.

Also note that it's actually `browser_time`.